recipe/meta.yaml

```
@@ -35,7 +35,7 @@ outputs:
         - liblapack
         - liblapacke
         - log4cxx
-        - minuit2
+        - minuit2_standalone
```

```suggestion
        - minuit2
```
 
Let's leave this as is and see if the new feedstock clears up the migration.